src/invoker.ts

```diff
@@ -100,7 +100,10 @@ export class ErrorHandler {
       sendCrashResponse({err, res: latestRes, callback: killInstance});
     });
 
-    process.on('exit', code => {
+    process.on('exit', (code: number | string) => {
+      if (typeof code === 'string') {
+        code = parseInt(code);
```

Comment:
should we change how we compare(line 110, to avoid crashing when code is string), than try to parse the string?

Reply:
it doesn't look like the comparison will cause crash either. Do you know where in our code it's crashing?

Reply:
The comparison isn't actually what's crashing -- it compares successfully but gets the value wrong since "0" === 0 is false. We'd like it to still exit silently when "0" (string zero) is provided.